src/containers/App.jsx

```diff
@@ -100,39 +101,43 @@ const App = ({
         height: '100%'
       }}
     >
-      <Corner />
-      <div
-        className="u-flex u-flex-column u-flex-content-start u-flex-content-stretch u-w-100 u-m-auto u-pos-relative"
-        ref={isReady ? div => setContentWrapper(div) : null}
-      >
-        <Alerter />
-        <MoveModal />
-        <HeroHeader />
-        {hasError && (
-          <Main className="main-loader">
-            <Failure errorType="initial" />
-          </Main>
-        )}
-        {isFetching && (
-          <Main className="main-loader">
-            <Spinner size="xxlarge" />
-          </Main>
-        )}
-        {isReady && (
-          <Switch>
-            <Route path="/redirect" component={IntentRedirect} />
-            <Route
-              path="/connected"
-              render={() => <Home base="/connected" wrapper={contentWrapper} />}
-            />
-            <Route exact path="/providers" component={StoreRedirection} />
-            <Route path="/providers/:category" component={StoreRedirection} />
-            <Redirect exact from="/" to="/connected" />
-            <Redirect from="*" to="/connected" />
-          </Switch>
-        )}
-        <IconSprite />
-      </div>
+      <MainView>
```

Comment:
why using another component? do we plan to reuse it somewhere else?

Reply:
Well, it seemed mandatory for the desired design to me. Also I prefer using named components even if used once instead of inlined html